templates/default/rdrs.service.erb

```diff
@@ -5,15 +5,26 @@ StartLimitIntervalSec=80
 StartLimitBurst=10
 
 [Service]
+
+<% if node['ndb']['rdrs']['containerize'].eql? "true" -%>
```

Comment:
Suggestion: generally it's safer to use `casecmp?` instead of `eql?` as the former is case insensitive.